shardingsphere-infra/shardingsphere-infra-context/src/test/java/org/apache/shardingsphere/infra/context/refresher/type/RenameTableStatementSchemaRefresherTest.java

```diff
@@ -0,0 +1,169 @@
+import org.apache.shardingsphere.infra.database.type.dialect.SQL92DatabaseType;
+import org.apache.shardingsphere.infra.eventbus.ShardingSphereEventBus;
+import org.apache.shardingsphere.infra.federation.optimizer.context.planner.OptimizerPlannerContext;
+import org.apache.shardingsphere.infra.federation.optimizer.metadata.FederationDatabaseMetaData;
+import org.apache.shardingsphere.infra.metadata.ShardingSphereMetaData;
+import org.apache.shardingsphere.infra.metadata.resource.ShardingSphereResource;
+import org.apache.shardingsphere.infra.metadata.rule.ShardingSphereRuleMetaData;
+import org.apache.shardingsphere.infra.metadata.schema.ShardingSphereSchema;
+import org.apache.shardingsphere.infra.metadata.schema.event.SchemaAlteredEvent;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.ddl.table.RenameTableDefinitionSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.generic.table.SimpleTableSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.generic.table.TableNameSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.RenameTableStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.value.identifier.IdentifierValue;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import javax.sql.DataSource;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public final class RenameTableStatementSchemaRefresherTest {
+
+    private final String oriTableNamePrefix = "TABLE";
+
+    private final String newTableNamePrefix = "NEW_TABLE";
+
+    private final String schemaMetaDataName = "SCHEMA_META_DATA_NAME";
+
+    private final String databaseName = "DATABASE_NAME";
+
+    private final String logicDataSourceName = "LOGIC_DATA_SOURCE_NAME";
+
+    @Mock
+    private ShardingSphereMetaData schemaMetaData;
+
+    @Mock
+    private FederationDatabaseMetaData database;
+
+    @Mock
+    private Map<String, OptimizerPlannerContext> optimizerPlanners;
+
+    private List<String> logicDataSourceNames;
+
+    @Mock
+    private RenameTableStatement sqlStatement;
+
+    @InjectMocks
+    private ConfigurationProperties props;
+
+    @Mock
+    private ShardingSphereResource shardingSphereResource;
+
+    @Mock
+    private ShardingSphereSchema shardingSphereSchema;
+
+    private RenameTableStatementSchemaRefresher renameTableStatementSchemaRefresher;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
`MockitoAnnotations.initMocks(this);` is unnecessary if `@RunWith(MockitoJUnitRunner.class)` existed.